src/Layout/Box/index.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { space } from 'styled-system';
+
+import { ScreenSize } from '../../Utils/StyleConfig';
+import { SpacingScaleValues } from '../Spacing';
+
+// Responsive
+type BreakpointAliases =
+  | 'default'
+  | 'ms'
+  | 'mm'
+  | 'ml'
+  | 't'
+  | 'ds'
+  | 'dm'
+  | 'dl';
+export const breakpointAliasMap = {
+  ms: `${ScreenSize.mobileS}px`,
+  mm: `${ScreenSize.mobileM}px`,
+  ml: `${ScreenSize.mobileL}px`,
+  t: `${ScreenSize.tablet}px`,
+  ds: `${ScreenSize.desktopS}px`,
+  dm: `${ScreenSize.desktopM}px`,
+  dl: `${ScreenSize.desktopL}px`,
+};
+const theme = {
+  breakpoints: breakpointAliasMap,
+};
+
+// Space
+type ResponsiveSpacing = {
```

Comment:
@rosslo , should we define all types in a separate file? Would it be more readable?